Gaijinizer patch/Map061.txt

```diff
@@ -4,62 +4,70 @@
 
  『E-003』の積み込み作業はどうした。
 > CONTEXT: Map061/events/5/pages/0/15/Dialogue/other2
-What are you doing. How was the loading operation
-of "E-003"?#MTLed
+What are you doing?
+
+Why aren't you loading E-003?#Ron
 > END STRING
 
 > BEGIN STRING
 ………研究員の皆さま方がやってるよ。
 
  手伝おうとしたら『素人は手を出すな』だとさ。
 > CONTEXT: Map061/events/5/pages/0/19/Dialogue/other2
-... All the researchers are doing.
-If you try to help, it is "Amateur should not put
-out a hand".#MTLed
+... The researchers are all workin' on it now.
+
+When I try to help, I get told "This isn't a job
+for amateurs".#Ron
 > END STRING
 
 > BEGIN STRING
 ま、俺としても事故って『E-003』に喰われたくねぇし
 
 言う通りにやつらの作業を待ってるよ。
 > CONTEXT: Map061/events/5/pages/0/23/Dialogue/other2
-Oh, I also want to get caught in the accident
-"E-003". I'm waiting for their work as I say.#MTLed
+Well not like I wanna get eaten by E-003 in some
+kinda incident, anyway.
+I'm doing what I'm told and waiting for them to
+finish up.#Ron
```

Comment:
A few changes for flow:
>_**Well, it's not like I wanna get eaten by E-003 in some
kinda "Incident", anyway.
I'm doing what I'm told and waiting for them to
finish up.**_